fedimint-testing/src/ln/real.rs

```diff
@@ -729,6 +743,13 @@ impl ILnRpcClient for LdkLightningTest {
     ) -> Result<EmptyResponse, LightningRpcError> {
         unimplemented!("Unsupported: we dont currently support HTLC interception for LDK Node");
     }
+
+    async fn update_fedimint_scids(
+        &self,
+        _scids: Vec<u64>,
+    ) -> Result<EmptyResponse, LightningRpcError> {
+        unimplemented!("Unsupported: we dont currently support updating scids for LDK Node");
```

Comment:
nit: should we return empty response for consistency between node impls where we don't support edge filtering?

Reply:
updated